tools/who_what_benchmark/whowhatbench/text2image_evaluator.py

```diff
@@ -26,6 +27,17 @@
 }
 
 
+class Generator(openvino_genai.Generator):
+    def __init__(self, seed, rng, mu=0.0, sigma=1.0):
+        openvino_genai.Generator.__init__(self)
+        self.mu = mu
+        self.sigma = sigma
+        self.rng = rng
+
+    def next(self):
+        return torch.normal(torch.tensor(self.mu), self.sigma, generator=self.rng)
```

Comment:
it is expected that generator returns torch tensor on genai side?

Reply:
it should return float number

Reply:
.item() needed here

`
>>> generator_out = torch.normal(torch.tensor(0.0), torch.tensor(1.0))
>>> generator_out
tensor(-0.6411)
>>> generator_out.item()
-0.6410552263259888
`

Reply:
thanks. I added but it does not have any effect on the generation results.